src/components/atoms/_OcTableRow/_OcTableRow.vue

```diff
@@ -1,12 +1,99 @@
 <template>
-  <tr tabindex="-1">
-    <slot />
+  <tr ref="observerTarget" tabindex="-1">
+    <oc-td v-if="isHidden" :colspan="lazyColspan">
+      <span class="shimmer" />
+    </oc-td>
+    <slot v-else />
   </tr>
 </template>
 <script>
+import { customRef, computed, ref } from "@vue/composition-api"
+import { useIsVisible } from "../../../composables"
+import OcTd from "../_OcTableCellData/_OcTableCellData.vue"
+
 export default {
   name: "OcTr",
   status: "ready",
   release: "1.0.0",
+  components: { OcTd },
+  props: {
+    lazy: {
+      type: Object,
+      required: false,
+      default: null,
+    },
+  },
+  setup(props) {
+    const observerTarget = customRef((track, trigger) => {
+      let $el
+      return {
+        get() {
+          track()
+          return $el
+        },
+        set(value) {
+          $el = value
+          trigger()
+        },
+      }
+    })
+
+    const lazyColspan = ref(null)
+    if (props.lazy) {
+      lazyColspan.value = props.lazy.colspan
+    }
+
+    const { isVisible } = props.lazy
+      ? useIsVisible({
+          ...props.lazy,
+          target: observerTarget,
+        })
+      : { isVisible: ref(true) }
+
+    const isHidden = computed(() => !isVisible.value)
```

Comment:
one more nitpick:
```suggestion
    const isHidden = computed(() => !unref(isVisible))
```
(and import `unref` in the composition api imports in line 10)